base_local_planner/src/local_planner_limits/__init__.py

```diff
@@ -35,6 +35,7 @@ def add_generic_localplanner_params(gen):
     gen.add("prune_plan", bool_t, 0, "Start following closest point of global plan, not first point (if different).", False)
 
     gen.add("xy_goal_tolerance", double_t, 0, "Within what maximum distance we consider the robot to be in goal", 0.1)
+    gen.add("inner_xy_goal_tolerance", double_t, 0, "Within what inner maximum distance we consider the robot to be in goal", 0.1)
```

Comment:
S: this is the text on PF?  I would explain that this is our best effort but not mandatory to succeed

Reply:
Fixed.